.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,69 @@
+          dotnet-version: '9.x'  
+
+      - name: Docker compose up
+        run: docker compose up -d && sleep 10
+
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v4
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)